canopen/node/remote.py

```diff
@@ -147,8 +147,16 @@ def __load_configuration_helper(self, index, subindex, name, value):
 
     def load_configuration(self):
         ''' Load the configuration of the node from the object dictionary.'''
+        # First apply PDO configuration from object dictionary
+        self.pdo.read(from_od=True)
+        self.pdo.save()
+
+        # Now apply all other records in object dictionary
         for obj in self.object_dictionary.values():
-            if isinstance(obj, ODRecord) or isinstance(obj, ODArray):
+            if 0x1400 <= obj.index < 0x1c00:
+                # Ignore PDO related objects
+                pass
+            elif isinstance(obj, ODRecord) or isinstance(obj, ODArray):
```

Comment:
```suggestion
                continue
            if isinstance(obj, ODRecord) or isinstance(obj, ODArray):
```

No need for else if you skip the iteration right away.